apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/index.jsx

```diff
@@ -383,14 +382,24 @@ const SwapForm = () => {
       targetCurrency: targetCurrency?.name,
       partner: provider,
     });
-    if (navigation) {
-      const { pathname, params } = navigation;
+
+    if (exchangeRate.providerType === "DEX") {
+      const from = swapTransaction.swap.from;
+      const fromAddress = from.parentAccount?.freshAddress || from.account.freshAddress;
+      const providerURL =
+        exchangeRate.providerURL || `/platform/${getProviderName(exchangeRate.provider)}`;
```

Comment:
If the provider is 1inch, `/platform/${getProviderName(exchangeRate.provider)}` will be `/platform/1inch`, whereas it looks like it used to be `/platform/1inch-lld` in `DEX_PROVIDERS`. 

Is this okay?

Reply:
Both are valid, I assume we are redirecting from "1inch" to "1inch-lld" in this case